lib/cli/src/commands/login.rs

```diff
@@ -237,38 +237,49 @@ impl Login {
         let auth_state = match self.token.clone() {
             Some(token) => Ok(AuthorizationState::TokenSuccess(token)),
             None => {
-                let person_wants_to_login =
-                    match wasmer_registry::whoami(env.dir(), Some(registry.as_str()), None) {
-                        std::result::Result::Ok((registry, user)) => {
-                            println!(
-                                "You are already logged in as {:?} on registry {:?}",
-                                user, registry
-                            );
-
-                            #[cfg(not(test))]
-                            {
-                                let login_again = Input::new()
-                                    .with_prompt(format!(
-                                        "{} {} - [y/{}]",
-                                        style("?").yellow().bold(),
-                                        style("Do you want to login with another user?")
-                                            .bright()
-                                            .bold(),
-                                        style("N").green().bold()
-                                    ))
-                                    .show_default(false)
-                                    .default(BoolPromptOptions::No)
-                                    .interact_text()?;
-
-                                login_again == BoolPromptOptions::Yes
-                            }
-                            #[cfg(test)]
-                            {
-                                false
-                            }
+                let person_wants_to_login = match wasmer_registry::current_user(
+                    env.dir(),
+                    Some(registry.as_str()),
+                    None,
+                ) {
+                    Result::Ok(Some(CurrentUser {
+                        registry,
+                        user,
+                        verified,
+                        ..
+                    })) => {
+                        println!(
+                            "You are already logged in as {:?} on registry {:?}",
+                            user, registry
+                        );
+                        if !verified {
+                            println!("Warning: Please verify your email address to fully access all Wasmer features");
```

Comment:
@Michael-F-Bryan If the person is not verified he can't create the tokens too. So he should not be able to login. And if he's not verified he also can't proceed with the login again path probably with that same account. In the case of person not being verified the login should just stop ?

Reply:
@Michael-F-Bryan Okay this thing has been redacted by @syrusakbary. Only warning will suffice

Reply:
As discussed, this behaviour is required and we want the person to be able to create tokens even not verified to improve the DX but show a warning like this.